--- main.js	2026-07-11 00:29:14
+++ main.js	2026-07-11 00:29:14
@@ -2307,6 +2307,11 @@
                     path: "/home",
                     name: "home",
                     label: "STREMIO_TV_NAV_HOME"
+                }, {
+                    path: "/anime",
+                    name: "anime",
+                    label: "Anime",
+                    icon: "tv-outline"
                 }, {
                     path: "/discover",
                     name: "discover",
