--- main.js	2026-07-11 00:29:14
+++ main.js	2026-07-11 00:29:14
@@ -5248,6 +5248,9 @@
             const b = (0, r.RZ)((() => n.e(5710).then(n.bind(n, 2181)))),
                 w = (0, r.RZ)((() => n.e(6415).then(n.bind(n, 2177)))),
                 k = (0, r.RZ)((() => n.e(8842).then(n.bind(n, 4781)))),
+                Anm = (0, r.RZ)((() => n.e(8842).then(n.bind(n, 4781)).then((e => ({
+                    default: e.X
+                }))))),
                 x = (0, r.RZ)((() => n.e(7962).then(n.bind(n, 1353)))),
                 S = (0, r.RZ)((() => n.e(7920).then(n.bind(n, 254)))),
                 A = (0, r.RZ)((() => n.e(6966).then(n.bind(n, 1912)))),
@@ -5292,6 +5295,11 @@
                             name: "discover",
                             path: "/discover/:transportUrl/:type/:id",
                             component: k,
+                            guard: t
+                        }, {
+                            name: "anime",
+                            path: "/anime",
+                            component: Anm,
                             guard: t
                         }, {
                             name: "library",
