--- video.chunk.js
+++ video.chunk.js
@@ -552,7 +552,7 @@
                                 }));
                             return t && n
                         })).catch((function() {
-                            return e.canPlayStream(t)
+                            return !0
                         }))
                     }))
                 }, t.manifest = {
--- 1477.chunk.js
+++ 1477.chunk.js
@@ -16417,7 +16417,7 @@
                                 }));
                             return t && n
                         })).catch((function() {
-                            return e.canPlayStream(t)
+                            return !0
                         }))
                     }))
                 }, t.manifest = {
